client/auth/login.jsx

```diff
@@ -180,6 +181,11 @@ export const Login = createReactClass( {
 			</Main>
 		);
 	},
+
+	handleChange( e ) {
```

Comment:
my gut said "shouldn't this be bound" but then I noticed this is a `createClass` class, not an es6 class. Just leaving this here as a warning to other reviewers.

Reply:
Yes, all components involved in this PR should be `createClass` classes. It's about removing a mixin, something that can't happen in a ES6 class.